### ISSUE TITLE
Dashboards: default overlay toggle state

### ISSUE THREAD

**Issue by realkosty** [getsentry/sentry#78068]
### Problem Statement

I would like the ability to have a "sticky" state for overlays visibility in Dashboards.

This is particularly helpful when using equations where the equation itself is the metric of interest while all other overlays are just intermediate steps to computing that metric.

Currently if I un-toggle intermediate metrics visibility to reduce clutter and see only the metric I care about (equation) it will get reset back to the original state once I navigate away from the dashboard or refresh it.

If I have a lot of widgets with lots of overlays (kind of the point of dashboards 🙂) it becomes impractical to un-toggle them every time 

### Solution Brainstorm

![Image](https://github.com/user-attachments/assets/e6743a06-d1ee-405d-8ff6-725e1a01c5f9)

### Product Area

Dashboards

**Comment by ale-cota** [comment]
Thanks, Kosty! That makes sense, adding this to our backlog.

**Comment by realkosty** [comment]
Thanks @ale-cota ! This one is minor and workaroundable. The real blocker for us is the default toggle state one: https://github.com/getsentry/sentry/issues/78068 

I realize it's more work but this unblocks are really compelling use case for enterprise customers and a highlight of our improved demo. We would like to take potential customers on journey from identifying a sudden drop in an end-to-end metric (cart->checkout) as a signal potentially pointing to errors or slowdowns introduced in latest release. This really connects the dots in terms of business value for budget holders (vs. just another obscure developer tool which lets you look at 1000s of likely irrelevant errors and perf data).

cc: @ndmanvar

**Comment by ale-cota** [comment]
> Thanks @ale-cota ! This one is minor and workaroundable. The real blocker for us is the default toggle state one: https://github.com/getsentry/sentry/issues/78068 

Kosty, you just referenced this same issue - is that intentional or did you mean to reference a different issue?

**Comment by realkosty** [comment]
@ale-cota I meant to leave that comment in https://github.com/getsentry/sentry/issues/78072 🤦‍♂️ 
This issue (https://github.com/getsentry/sentry/issues/78068) is most important for us.

**Comment by ale-cota** [comment]
OK, got it 👍 we'll hopefully be able to look at this one soon.

**Comment by thinkocapo** [comment]
I have the same problem. Another sub-problem in this is that the y-axis isn't accurate because it has to account for multiple overlays. So if you look at the y-axis first, it really means nothing:
![Image](https://github.com/user-attachments/assets/99303092-321b-47cb-bc08-9a95c8c3a231)
but I only need this one:
![Image](https://github.com/user-attachments/assets/d639dba1-8e6d-46f4-b167-a2ba387a5864)
[dashboard link](https://demo.sentry.io/dashboard/54856/?project=5808623&statsPeriod=30d)

**Comment by ale-cota** [comment]
@thinkocapo, @realkosty You can actually delete the "intermediate" overlays if you only ever care to display the result of the equation. The equation will continue to work.

https://github.com/user-attachments/assets/2aee8910-e0cf-4aea-ac30-e8bb85ad4b77

We'll still look into potentially improving the discoverability on this, but does this solve what you need here? Thanks.

**Comment by realkosty** [comment]
@ale-cota this is fantastic! 
I did not realize that 😄 This reduces the urgency of this issue for sure.

Still would be nice to have in case someone wants an easy way to look at absolute numbers behind the equation metrie.

**Comment by realkosty** [comment]
@thinkocapo i wonder if that should be a separate issue - "automatic multiple y-axes when displaying multiple metrics with very different ranges of values"?

**Comment by ale-cota** [comment]
@thinkocapo yes, having multiple y-axis would definitely be a separate issue, that's not related only to equations.
@realkosty I would then close this issue and we'll address the better UX for equations in the query builder redesign 👍
